mcda/utils/application_enums.py

```diff
@@ -0,0 +1,40 @@
+from enum import Enum
```

Comment:
This file lists all the Enumeration objects so that Configuration model can be accessed using enum values without worrying about spelling mistakes, cases, etc when typing keys.
Please refer [this stackoverflow comment](https://stackoverflow.com/a/37601645) for more detailed explanation for why it should be used.

I would leave the name of the file as it is because it appropriately describes that all the enums are contained in this file and also leaving scope open for further addition of different enums.